ui/src/main/webapp/app/query/query-component.js

```diff
@@ -201,9 +201,16 @@ function QueryController($scope, queryPage, operationService, types, graph, conf
             } else {
                 submitResults(data);
             }
-        }), function(err) {
+        }, function(err) {
             loading.finish();
-        };
+            var errorString = 'Error executing operation';
```

Comment:
Perhaps we should have an error service that has this common code in. Then in the future we could handle errors in a better way rather than using alerts.

Reply:
I was thinking that

Reply:
I have created an issue for this #300